site/using-cluster-operator.md

```diff
@@ -0,0 +1,790 @@
+This field shows the Service name and namespace. The following command shows how to fetch this information:
+
+<pre class='hljs lang-bash'>
+kubectl get rabbitmqcluster INSTANCE \
+-ojsonpath='Name: {.status.admin.serviceReference.name} -- Namespace: {.status.admin.serviceReference.namespace}'
+</pre>
+
+Where `INSTANCE` is the name of `RabbitmqCluster` resource.
+
+For more information on how to connect using Services, check
+[Kubernetes documentation](https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/#services)
+regarding Service DNS.
+
+### <a id='creds' class='anchor' href='#creds'>Retrieve Your RabbitMQ Admin Credentials</a>
+
+Admin credentials for a RabbitmqCluster are stored in a Kubernetes secret called `INSTANCE-rabbitmq-admin`,
+where `INSTANCE` is the name of your `RabbitmqCluster` object.
+Kubernetes encodes secrets using base64.
+
+The name and namespace of the secret is also present in the custom resource status. To retrieve the Secret name,
+run `kubectl get rabbitmqcluster INSTANCE -ojsonpath='{.status.admin.secretReference.name}'`
+
+To retrieve credentials and display them in plaintext:
+
+1. Display the username by running:
+
+    ```
+    kubectl -n NAMESPACE get secret INSTANCE-rabbitmq-admin -o jsonpath="{.data.username}" | base64 --decode
+    ```
+
+    Where:
+    <ul>
+      <li><code>INSTANCE</code> is the name of your <code>RabbitmqCluster</code></li>
+      <li><code>NAMESPACE</code> is the Kubernetes namespace that contains <code>RabbitmqCluster</code></li>
+    </ul>
+
+1. Display the password by running:
+
+    ```
+    kubectl -n NAMESPACE get secret INSTANCE-rabbitmq-admin -o jsonpath="{.data.password}" | base64 --decode
+    ```
+
+## <a id='verify-instance' class='anchor' href='#verify-instance'>Verify the Instance is Running</a>
+
+Deploy the RabbitMQ throughput testing tool PerfTest to quickly verify that your instance is running correctly.
+For more information, see [PerfTest](https://github.com/rabbitmq/rabbitmq-perf-test) in GitHub.
+
+<p class="note">
+  <strong>Note:</strong> If you run the below commands from outside the RabbitmqCluster namespace, add
+  <code>-n NAMESPACE</code> to the kubectl commands below.
+</p>
+
+1. To install and run PerfTest, run these commands:
+
+<pre class="terminal">
+    instance=INSTANCE-NAME
+    username=$(kubectl get secret ${instance}-rabbitmq-admin -o jsonpath="{.data.username}" | base64 --decode)
+    password=$(kubectl get secret ${instance}-rabbitmq-admin -o jsonpath="{.data.password}" | base64 --decode)
+    service=${instance}-rabbitmq-ingress
+    kubectl run perf-test --image=pivotalrabbitmq/perf-test -- --uri "amqp://${username}:${password}@${service}"
+</pre>
+
+1. Verify that PerfTest is sending and receiving messages by running:
+
+    ```
+    kubectl logs -f perf-test
+    ```
+
+    A log appears as in this example:
+
+  <pre class="hljs terminal">
```

Comment:
We usually would format this as

``` shell

kubectl logs -f perf-test
# =>  id: test-104555-858, starting consumer #0
# =>   id: test-104555-858, starting consumer #0, channel #0
# =>   id: test-104555-858, starting producer #0
# =>   id: test-104555-858, starting producer #0, channel #0
# =>   id: test-104555-858, time: 1.000s, sent: 19057 msg/s, received: 11768 msg/s, min/median/75th/95th/99th consumer latency: 4042/140608/190841/251618/258979 micro-s
# =>   id: test-104555-858, time: 2.000s, sent: 24020 msg/s, received: 16283 msg/s, min/median/75th/95th/99th consumer latency: 222998/507432/642110/754038/776600 micro-s
```

or similar. The use of `$` to indicate a shell has been eradicated a couple of years ago.